src/packages/Controls/LayerSwitcher/LayerSwitcher.js

```diff
@@ -1273,6 +1276,126 @@ var LayerSwitcher = class LayerSwitcher extends Control {
         logger.debug(e);
     }
 
+    /**
+     * togglegreyscale
+     * @param {Event} e - Event
+     * @private
+     */
+    _onToggleLayerGreyscaleClick (e) {
```

Comment:
US or English, you have to make a choice !

Reply:
Réparé